### ISSUE TITLE
Fix buttons throughout the tool/make consistent

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#548]
o	Fix buttons same gray color (will be based on the gray color and button example that UCOMM will provide)
o	Fix button text size and weight/thickness equal throughout (will be based on the button example that UCOMM will provide)
o	Fix button edges the same (Will be based on the edge of the button example that UCOMM will provide)
o	Fix button effects the same (UCOMM requested that all drop shadow effect should be removed – will be based on the button example that UCOMM will provide)
o	Use consistent singular/plural language within the button text

**Comment by caadler** [comment]
Buttons have been updated as individual pages have been updated.  Future updates will be made on separate tickets.
